stories/topics/proc-aap-aws-backup-and-recovery.adoc

```diff
@@ -0,0 +1,68 @@
+
+=== Modify existing CloudFormation Template to reference restored EFS
+. Search for `storage6B604F96`
+. Remove the entire filesystem component `storage6B604F96`
+. In the JSON template under `Resources` > `storagemounttarget1A24DC9FD` > `Properties` update the value for `FileSystemId` to the filesystem ID of the restored EFS (recorded in the steps above)
+. In the JSON template under `Resources` > `storagemounttarget273289903` > `Properties` update the value for `FileSystemId` to the filesystem ID of the restored EFS (recorded in the steps above)
+. In the JSON template under `Resources` > `storageaccesspoint6DF311E5` > `Properties` update the value for `FileSystemId` to the filesystem ID of the restored EFS (recorded in the steps above)
+. In the JSON template under `Resources` > `controllerautoscalegroupLaunchConfig02845540` > `Properties` > `UserData` > `Fn::Base64` > `Fn::Join[1][0]` after `EFS_VOUME=` add the filesystem ID of the restored EFS (recorded in the steps above) and remove the object directly below this line (contains `Fn::GetAtt`)
+. In the JSON template under `Resources` > `controllerautoscalegroupASG89CA6D06` > `Metadata` > `AWS::CloudFomration::Init` > `copyBootstrap` > `files` > `/bootstrap_node.sh` > `content` > `Fn::Join[1][0]` after `EFS_VOUME=` add the filesystem ID of the restored EFS (recorded in the steps above) and remove the object directly below this line (contains `Fn::GetAtt`)
+. In the JSON template under `Resources` > `hubautoscalegroupLaunchConfig3FD9231D` > `Properties` > `UserData` > `Fn::Base64` > `Fn::Join[1][0]` after `EFS_VOUME=` add the filesystem ID of the restored EFS (recorded in the steps above) and remove the object directly below this line (contains `Fn::GetAtt`)
+. In the JSON template under `Resources` > `hubautoscalegroupASG7FC5A5DF` > `Metadata` > `AWS::CloudFomration::Init` > `copyBootstrap` > `files` > `/bootstrap_node.sh` > `content` > `Fn::Join[1][0]` after `EFS_VOUME=` add the filesystem ID of the restored EFS (recorded in the steps above) and remove the object directly below this line (contains `Fn::GetAtt`)
+. In the JSON template under `Outputs` > `storageEFSID4982FF06` update the value for `Value` to `arn:aws:elasticfilesystem:<region>:<account_id>:file-system/<restored_efs_id>`
+. Save this file
+
+
+=== Create a new AAP Deployment Using the Modified Template
+. In the AWS Console, navigate to *CloudFormation* and click on btn:[Create Stack] located near the top right of the screen
+. Select *With new resources (standard)* in the dropdown
+. In the *Specify template* section under *Template source* select btn:[Upload a template file] and select the template you have modified. Click btn:[Next] to go to the next step
```

Comment:
select *Upload a template file*

. Click btn:[Next] to go to the next step.  <This should be on a separate line>